src/jms2.md

```diff
@@ -25,11 +25,13 @@ User feedback is welcome! Please comment on the JIRAs with questions and comment
 
 JIRA|Target Version|Completed Version|Feature|Notes
 ---|---|---
-[AMQ-7309](https://issues.apache.org/jira/browse/AMQ-7309) | 5.17.0.RC1 | | JMS v2.0 API dependency | ActiveMQ will ship with a JMS v2.0 dependency jar
-[AMQ-8320](https://issues.apache.org/jira/browse/AMQ-8320) | 5.17.0.RC1 | | Delivery Delay | Support for Message DeliveryDelay feature
-[AMQ-8321](https://issues.apache.org/jira/browse/AMQ-8321) | 5.17.0.RC1 | | GetBody/isBodyAssignable | Support for checking body type using a Class<?>
-[AMQ-8322](https://issues.apache.org/jira/browse/AMQ-8322) | TBD | | Connection.createContext | Simplified JMS Context object support
-[AMQ-8323](https://issues.apache.org/jira/browse/AMQ-8323) | TBD | | Session Topic Consumer | Consume messages using a MessageConsumer object (same as Queue)
-[AMQ-8324](https://issues.apache.org/jira/browse/AMQ-8324) | 5.17.0.RC1 | | MessageProducer send methods | Completion Listener and Delivery Delay support
+[AMQ-7309](https://issues.apache.org/jira/browse/AMQ-7309) | 5.17.0 | | JMS v2.0 API dependency | ActiveMQ will ship with a JMS v2.0 dependency jar
+[AMQ-8322](https://issues.apache.org/jira/browse/AMQ-8322) | 5.17.0 | | Connection.createContext | Simplified JMS Context object support
+[AMQ-8322](https://issues.apache.org/jira/browse/AMQ-8322) | 5.17.0 | | Session Topic Consumer | Consume messages using a MessageConsumer object (same as Queue)
+[AMQ-8322](https://issues.apache.org/jira/browse/AMQ-8322) | 5.17.0 | | MessageProducer send methods | 
+[AMQ-8320](https://issues.apache.org/jira/browse/AMQ-8320) | 5.17.1 | | Delivery Delay | Support for Message DeliveryDelay feature
+[AMQ-8321](https://issues.apache.org/jira/browse/AMQ-8321) | 5.17.1 | | GetBody/isBodyAssignable | Support for checking body type using a Class<?>
+[AMQ-8324](https://issues.apache.org/jira/browse/AMQ-8324) | TBD | | MessageProducer send methods | Completion Listener and Delivery Delay support
```

Comment:
First "MessageProducer send methods" line has a terminator when no other does.

Seems odd to have 2 lines and JIRAs referenced for "MessageProducer send methods" as a feature, when the first is unqualified with notes. If its incomplete it should probably be noted in the first instance. Especially as the second has no timeline given and its notes mention the main send feature additions in JMS 2.

Reply:
I re-worked the table to clarify the intent